File: exercise-08-pizza-party/PizzaParty.java
/**
 * A program to evenly divide pizzas.
 * Exercise 8 Pizza Party, Exercises for Programmers by Brian Hogan
 *
 * @author James Schmidt
*/
class PizzaParty {
	public static void main(String[] args) {
		String input = System.console().readLine("How many people? ");
		int people = Integer.parseInt(input);

		input = System.console().readLine("How many pizzas do you have? ");
		int pizzas = Integer.parseInt(input);

		input = System.console().readLine("How many slices per pizza? ");
		int slices = Integer.parseInt(input);

		int slicesEach = (pizzas * slices) / people;
		int remainder = (pizzas * slices) % people;

		System.out.format("%n%d people with %d pizza%n", people, pizzas);
		System.out.format("Each person gets %d piece of pizza.%n", slicesEach);
		System.out.format("There are %d leftover pieces.%n", remainder);
	}
}

File: exercise-08-pizza-party/pizza_party.py
PIECES_PER_PIZZA = 8

people = int(input("How many people? "))
pizzas = int(input("How many pizzas do you have? "))

total_pieces = pizzas * PIECES_PER_PIZZA
pieces_each = total_pieces // people
leftover_pieces = total_pieces % people

print()
print(f"{people} people with {pizzas} pizzas")
print(f"Each person gets {pieces_each} pieces of pizza.")
print(f"There are {leftover_pieces} leftover pieces.")
